models/reviews-models.js

```diff
@@ -2,14 +2,70 @@ const connection = require("../db/connection");
 
 exports.fetchReviewByReviewId = (id) => {
   if (typeof Number(id) === "number" && !isNaN(Number(id))) {
-    if (Number(id) > 0 && Number(id) < 14) {
-      return connection
-        .query(`SELECT * FROM reviews WHERE review_id = $1;`, [id])
-        .then((review) => {
+    return connection
+      .query(
+        `SELECT a.*, COUNT (b.review_id) AS comment_count FROM reviews a
+        LEFT JOIN comments b
+        ON a.review_id = b.review_id
+        WHERE a.review_id = $1
+        GROUP BY a.review_id;`,
```

Comment:
Is there a reason you've changed the readable table names for a generic `a` and `b`?